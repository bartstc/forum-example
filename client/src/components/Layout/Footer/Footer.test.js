import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe('Footer', () => {

  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('renders Footer without errors', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

});