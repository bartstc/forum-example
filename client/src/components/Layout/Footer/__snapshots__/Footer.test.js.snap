// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer without errors 1`] = `
<footer
  className="bg-dark text-white mt-5 p-4 text-center"
>
  Copyright Â© 
  2019
   Forum example
</footer>
`;
